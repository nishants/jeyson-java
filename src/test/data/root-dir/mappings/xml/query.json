{
  "request": {
    "method": "GET",
    "urlPattern": "/xml/query?.*"
  },
  "response": {
    "status": 200,
    "bodyFileName": "xml/query.xml"
  }
}