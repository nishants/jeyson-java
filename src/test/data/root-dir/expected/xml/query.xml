<data><queries><time>now</time><message>hi</message></queries></data>